pmpro-sponsored-members.php

```diff
@@ -1767,6 +1817,70 @@ function pmprosm_display_sponsored_accounts($member_ids) {
 	ob_end_clean();
 	return $content;
 }
+function pmprosm_remove_member_from_seat($member_id, $levelstocancel, $user_id){ 
+    /*
+     * Double checking user_id is a sponsor
+     */
+    $user_level = pmpro_getMembershipLevelForUser($user_id);
+    if( pmprosm_isSponsoredLevel($user_level->id) )
+    {
+        /*
+         * Removes member from seat
+         */
+        $response = pmprosm_remove_member_from_seat_action($user_id, $member_id, $levelstocancel);
+
+        /*
+         * Get Updated member_ids
+         */
+        if($response){        
+            $member = get_userdata($member_id);
+            echo '<div style="margin: 0 auto 15px; padding: 20px; color: #f00; border: solid 1px #f00;">Sponsored User: '.$member->display_name.' was removed. (Membership Level: '.$levelstocancel.')</div>';

```

Comment:
Is there a class we can use instead of the style here?

Can update to localize the string. Echo the HTML separate. Then use sprintf and _e to echo the strings with placeholders.

Reply:
I hope my update here is what you were looking for.